### PR TITLE
Implement the bottom action toolbar - inventories [SCI-8299]

### PR DESCRIPTION
**Pull request by G-Chubinidze**
Jira ticket: [SCI-8299](https://scinote.atlassian.net/browse/SCI-8299)

### What was done
_bottom action toolbar for inventories_

[SCI-8299]: https://scinote.atlassian.net/browse/SCI-8299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by artoscinote — CHANGES_REQUESTED** [review]
Fix linter issues.

File: app/assets/stylesheets/shared/datatable.scss [hunk 1]
```diff
@@ -38,11 +38,16 @@
...
-    min-height: var(--datatable-pagination-row);
-    position: relative;
-    width: 100%;
+    height: 5em;
+    left: var(--left-navigation-width);
+    padding: 0 2em;
+    position: fixed;
+    transition: .4s $timing-function-sharp;
+    width: calc(100% - var(--left-navigation-width));
+    z-index: 90;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/shared/datatable.scss:50]: this not correct, rebase on develop must not be fixed.

File: app/assets/javascripts/repositories/index.js [hunk 2]
```diff
@@ -143,7 +151,7 @@
     });
   }
 
-  $('.repositories-index')
+  $(document)
```
Comment (aignatov-bio) [inline app/assets/javascripts/repositories/index.js:154]: don't use document

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

**Review by aignatov-bio — CHANGES_REQUESTED** [review]
Fix hound

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**